modules/MagnetometerCalculator.py

```diff
@@ -41,7 +41,7 @@ def _Response_funtion(self,freq):
         I = 1.5
         gama = gs*self.BhorMagneticMoment/(self.ReductionPlankConstant*(2*I+1))
         ### calculate the complex response value 
-        output = self.T2*gama/(-2*1j+2*self.T2*2*np.pi*(self.consider_freq-freq))
+        output = 150*self.T2*gama/(-2*1j+2*self.T2*2*np.pi*(self.consider_freq-freq))
```

Comment:
It would be better to have some comments written if you hardcode some number here. To keep a record and to make it clear to the other code developers.

Reply:
in changqing's code,  there is a 150 times magnification in this response function, i add it into our code